e2e_samples/fabric_dataops_sample/infra/setup-infra.sh

```diff
@@ -1,257 +1,65 @@
 #!/bin/bash
-source .env
+
+#######################################################
+# Deploys all necessary azure and Fabric resources
+#
+# Prerequisites:
+# - User is logged in to the azure cli
+# - Correct Azure subscription is selected
+#######################################################
+
+source ./.env
 
 ## Environment variables
-base_name="$BASE_NAME"
-location="$LOCATION"
-adls_gen2_connection_id="$ALDS_GEN2_CONNECTION_ID"
+base_name="$TF_VAR_base_name"
```

Comment:
Make the variable naming consistent with .env file. I suggest "BASE_NAME". Other variables required by this script are:

LOCATION
FABRIC_TENANT_ID
# Skip the below variables as the GIT integration APIs don't have SP support yet. But eventually these would be required.
ORGANIZATION_NAME
PROJECT_NAME
REPOSITORY_NAME
BRANCH_NAME
DIRECTORY_NAME

Reply:
@promisinganuj please review the newly committed `.envtemplate` and let me know